docs/user_guide/emummc/partitioning_sd.md

```diff
@@ -16,20 +16,20 @@ What you need:
 
 ### Instructions
 
-1. Inject the TegraExplorer payload with your 64GB (or larger) SD card inserted into your Switch.
+1. Inject the TegraExplorer payload with your 64GB (or larger) microSD Card inserted into your Switch.
 	- If you forgot how to do this, re-read the [sending payload](sending_payload.md) section of the guide.
-2. Navigate to `Partition the sd` and press the A button to enter the SD format menu.
-	- If you can't find the Partition the sd option, make sure that your SD card is inserted and select Mount SD.
-3. Navigate to `Fat32 + EmuMMC` and press the A button to confirm.
-4. Read the warning, and select Yes to partition & format your SD card.
-	- Note: This will delete all data on your SD card. Make sure you backed up your Nintendo folder!
+2. Navigate to `Partition the SD` and press the A button to enter the SD format menu.
+	- If you can't find the Partition the sd option, make sure that your microSD Card is inserted and select Mount SD.
+3. Navigate to `FAT32 + emuMMC` and press the A button to confirm.
```

Comment:
It is actually `Fat32 + EmuMMC`

Reply:
Also I think the sd in partition the SD in the menu is small (@ meme smh)